README.md

```diff
@@ -32,7 +32,12 @@ This micro-service recognizes the following messages:
   * _role:github,cmd:query_
   * _role:github,cmd:parse_
 
+## Running with Curl
 
+Any of the messages above can be run using curl in the following format
+```
+curl -d '{"role":"github","cmd":"get"}' http://localhost:52472/act
```

Comment:
I found it by editing the mesh.js to print the port number, but I don't think that would be reasonable to expect an end user to do